### PR TITLE
Bump version to 0.18

### PR DESCRIPTION
**Pull request by maksbotan**

### CODE REVIEW

**Comment by maksbotan** [comment]
Follow-up in `servant-quickcheck`: https://github.com/haskell-servant/servant-quickcheck/pull/72

Once both these PRs are merged and `servant-quickcheck` is released to Hackage, we can re-enable `cookbook/testing` here.

CC @fisx.

File: .travis.yml [hunk 1]
```diff
@@ -195,6 +195,10 @@ install:
     echo "constraints:  sqlite-simple < 0"                       >> cabal.project
     echo "allow-newer:  servant-pagination-2.2.2:servant"        >> cabal.project
     echo "allow-newer:  servant-pagination-2.2.2:servant-server" >> cabal.project
+    echo "allow-newer:  servant-multipart:servant"               >> cabal.project
+    echo "allow-newer:  servant-multipart:servant-server"        >> cabal.project
+    echo "allow-newer:  servant-multipart:servant-client-core"   >> cabal.project
+    echo "allow-newer:  servant-js:servant"                      >> cabal.project
```
Comment (fisx) [inline .travis.yml:201]: note to self: remove this again when everything is upgraded!  (also further down in this file and in cabal.project)

**Comment by fisx** [comment]
thanks!

**Comment by maksbotan** [comment]
Thanks! `servant-multipart` is already merged, `servant-quickcheck` has a PR. Now we only need Hackage releases for both :)

In `servant-quickcheck` PR I did not bump the version, though. I think you should do it after you merge it.

**Comment by fisx** [comment]
> Thanks! `servant-multipart` is already merged, `servant-quickcheck` has a PR. Now we only need Hackage releases for both :)
> 
> In `servant-quickcheck` PR I did not bump the version, though. I think you should do it after you merge it.

http://hackage.haskell.org/package/servant-quickcheck-0.0.10.0

(I'll wait a little longer for the docs to regenerate, if that doesn't happen I'll upload them.)

**Pull request merged**